iwyu_output.cc

```diff
@@ -2204,6 +2214,14 @@ void IwyuFileInfo::ResolvePendingAnalysis() {
       }
     }
   }
+
+  for (NamespaceDeclUse& use : namespace_uses) {
+    const NamespaceDecl* decl = GetBestNamespaceDecl(use.decl);
+    if (decl != nullptr) {
+      // For each namespace use, report against the best NamespaceDecl
```

Comment:
I'd move this comment to just before the loop. Also, add a full stop to end the sentence.